api/HTMLInputElement.json

```diff
@@ -1050,7 +1050,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": true
+              "version_added": "13"
```

Comment:
`selectionDirection` confirmed missing in Edge 12. `selectionStart` is there though, so I trust this.